public/js/components/EditorSearchBar.js

```diff
@@ -126,7 +126,7 @@ const EditorSearchBar = React.createClass({
 
   render() {
     if (!isEnabled("search") || !this.state.enabled) {
-      return dom.div();
+      return null;
```

Comment:
Returning null is nicer as it's easier to work around in CSS than invisible divs.